### PR TITLE
Release 4.5.0rc0

### PR DESCRIPTION
**Pull request by muhammadhamzasajjad**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by muhammadhamzasajjad** [comment]
@conda-forge-admin, please rerender

**Comment by muhammadhamzasajjad** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -109,6 +109,7 @@ requirements:
     - protobuf >=3.5.1, <5
     - lmdb
     - packaging
+    - trustme
```
Comment (jamesmunro) [inline recipe/meta.yaml:112]: Why is `trustme` in the `run` and not the `test` section?

**Comment by muhammadhamzasajjad** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml
Comment (jjerphan) [inline recipe/meta.yaml:1]: Reason for the error on OSX:

- https://github.com/man-group/ArcticDB/pull/867 which is part of `4.5.0rc0` [requires `azure-core-cpp==1.11.x`](https://github.com/man-group/ArcticDB/commit/5dd6a642c3ab70be0ead45665c8cfcfa8cffafa3#diff-0170a0485b340c2d8c22a7b86c1f2702e1405aef34b69d6930be9cbc9413a782R73)

 - Currently [`azure-core-cpp==1.10.2` is picked](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=961288&view=logs&j=9c5ef928-2cd6-52e5-dbe6-9d173a7d951b&t=20c71c51-4b27-578b-485d-06ade2de1d00&l=1032)
 
Solution:
 - pin `azure-core-cpp>=1.11.0`

Related: https://github.com/conda-forge/arcticdb-feedstock/pull/232/.

**Comment by muhammadhamzasajjad** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 2]
```diff
@@ -68,7 +68,7 @@ requirements:
     - gflags
     - msgpack-c
     - aws-sdk-cpp
-    - azure-core-cpp
+    - azure-core-cpp >=1.11.0
```
Comment (jjerphan) [inline recipe/meta.yaml:71]: Currently [this overwrites](https://github.com/conda-forge/arcticdb-feedstock/pull/233/files#diff-c399e81833f8217983f67b22ce193230d00ffae438a62252a1538fb602479cb3L7-L8) [those specifications](https://github.com/conda-forge/arcticdb-feedstock/blob/rc/recipe/conda_build_config.yaml#L15-L39).

For some more precision, could this pin be ported [here](https://github.com/conda-forge/arcticdb-feedstock/blob/rc/recipe/conda_build_config.yaml#L15-L39) not to overwrite them with this comment:

```suggestion
    # Pinned to >=1.11.0 for the `main` specification
    # See `./conda_build_config.yaml`.
    - azure-core-cpp
```

?

**Comment by muhammadhamzasajjad** [comment]
@conda-forge-admin, please rerender

File: recipe/conda_build_config.yaml [hunk 3]
```diff
@@ -23,7 +23,7 @@ aws_sdk_cpp:
   - "1.10.57"
   - "*"
 azure_core_cpp:
-  - "1.10.2"
+  - "1.11.0"
   - "*"
```
Comment (jjerphan) [inline recipe/conda_build_config.yaml:27]: My previous remark was not clear enough.
```suggestion
  - "1.10.2"
  - ">=1.11.0""
```

**Comment by muhammadhamzasajjad** [comment]
@conda-forge-admin, please rerender

**Comment by muhammadhamzasajjad** [comment]
@conda-forge-admin, please rerender

File: recipe/conda_build_config.yaml (hunk 3 above)
Comment (muhammadhamzasajjad) [reply recipe/conda_build_config.yaml]: It is still installing the older version [here](https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=961430&view=logs&j=9c5ef928-2cd6-52e5-dbe6-9d173a7d951b&t=20c71c51-4b27-578b-485d-06ade2de1d00&l=1032)

Comment (jjerphan) [reply recipe/conda_build_config.yaml]: Yes, this is normal and expected for the `libevent-2.1.0` specification.

Comment (jjerphan) [reply recipe/conda_build_config.yaml]: I think we have two options: (temporary) skip those builds, leaving them untouched, or overwriting these specifications.

I would go for the first one.

What do you think?

Comment (muhammadhamzasajjad) [reply recipe/conda_build_config.yaml]: How will the package be published if we don't run the builds?

Comment (muhammadhamzasajjad) [reply recipe/conda_build_config.yaml]: I am fine with it if the RC can be published.

Comment (jjerphan) [reply recipe/conda_build_config.yaml]: 4.5.0rc0 can be released but we can skip some the problematic ones.

Comment (muhammadhamzasajjad) [reply recipe/conda_build_config.yaml]: OK. Then. Let's do that.

**Comment by muhammadhamzasajjad** [comment]
@conda-forge-admin, please rerender

**Comment by muhammadhamzasajjad** [comment]
@conda-forge-admin, please rerender

**Comment by jjerphan** [comment]
@conda-forge-admin, please rerender

**Comment by jjerphan** [comment]
@conda-forge-admin, please rerender

**Comment by jjerphan** [comment]
@conda-forge-admin, please rerender

**Comment by jjerphan** [comment]
Closing since #242 has been merged.

**Pull request closed without merging**